data/rescue/Austin1998/Austin1998_Table4.csv

```diff
@@ -0,0 +1,127 @@
+"Leaf mass per area (LMA), foliar nitrogen, phosphorus, major cations, δ^{15}N, and δ^{13}C for native overstory species from sites along the precipitation gradient in Hawai'i. All concentrations are percent dry weight, and values are means with SE in parentheses. Where no SE is reported, a single or composite sample was analyzed. Species names from @Wagner1990.",,,,,
+-,Kaniku,Kaupulehu,Manuka,Kaloko,Waiakea
+Species,LMA,LMA,LMA,LMA,LMA

```

Comment:
You have serveral tables here embeded in one table (reflecting the paper). These need to be restructured for one header. This requires restructuring the table. I suggest keeping the first header on line 2 since that is uniform across the subtables and rotating the species into a new column. Come talk with me if you have quesitons